pkg/kubectl/cmd/create.go

```diff
@@ -97,11 +97,11 @@ func RunCreate(f *cmdutil.Factory, out io.Writer, filenames util.StringList) err
 	err = r.Visit(func(info *resource.Info) error {
 		data, err := info.Mapping.Codec.Encode(info.Object)
 		if err != nil {
-			return err
+			return fmt.Errorf("error when creating %q: %v ", info.Source, err)
```

Comment:
Technically anyone could call it, but in practice it's only ever called in NewCmdCreate and in tests. Come to think of it, all the Run* functions should actually be private.